### PR TITLE
Adjust top-up modal

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

**Comment by JustCaptcha** [comment]
image

image

image

image

**Comment by JustCaptcha** [comment]
image

image

**Comment by JustCaptcha** [comment]
image

image

image

image

**Comment by JustCaptcha** [comment]
image

image

File: universal-login-react/src/ui/styles/themes/Jarvis/chooseTopUpThemeJarvis.sass [hunk 1]
```diff
@@ -35,6 +39,7 @@
     border: 1px solid #E1E1F0
     border-radius: 0
 
+
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/themes/Jarvis/chooseTopUpThemeJarvis.sass:42]:
```suggestion
```

**Pull request merged**